apps/api_web/test/api_web/views/trip_view_test.exs

```diff
@@ -55,4 +76,19 @@ defmodule ApiWeb.TripViewTest do
     rendered = render(ApiWeb.TripView, "index.json-api", data: @trip, conn: conn)
     refute rendered["data"]["relationships"]["vehicle"] == nil
   end
+
+  test "doesn't include stops if not explicitly included", %{conn: conn} do
+    rendered = render(ApiWeb.TripView, "index.json-api", data: @trip, conn: conn)
+    assert rendered["data"]["relationships"]["stops"] == nil
+  end
+
+  test "render includes stop list if explicitly included", %{conn: conn} do
+    conn =
+      conn
+      |> Map.put(:params, %{"include" => "stops"})
+      |> ApiWeb.ApiControllerHelpers.split_include([])
+
+    rendered = render(ApiWeb.TripView, "index.json-api", data: @trip, conn: conn)
+    refute rendered["data"]["relationships"]["stops"] == nil
```

Comment:
Try an assertion like this:
```elixir
assert %{"data" => [_ | _]} = rendered["data"]["relationships"]["stops"]
```
It doesn't look like your new code is actually being called.

Reply:
Fixed